test/AutomationNotes.MD

```diff
@@ -0,0 +1,38 @@
+# AUTOMATION NOTES:
```

Comment:
Right now, everything we have wrt development is simply in CONTRIBUTING.md. We're talking about reorganizing our development docs, and this will likely happen in the nearish future, at which point we'll have a better place to put docs for specific topics, such as these.

I think for now, in the interest of being consistent (until we can try to make it better), we should put all of this in CONTRIBUTING.md as just another section, possibly under a Testing/Functional Testing header.